oak-segment-azure/src/test/resources/logback-test.xml

```diff
@@ -21,11 +21,19 @@
             <pattern>%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
         </encoder>
     </appender>
-    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener"/>
```

Comment:
why do we need these changes for a change in DocumentStore?

Reply:
space re-added in https://github.com/apache/jackrabbit-oak/pull/1459